package.json

```diff
@@ -24,9 +24,9 @@
   ],
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
-    "vscode-chrome-debug-core": "^4.2.2",
-    "vscode-debugadapter": "^1.29.0",
-    "vscode-nls": "^3.2.1"
+    "vscode-chrome-debug-core": "^5.0.2",
+    "vscode-debugadapter": "^1.28.0",
```

Comment:
Any reason this is changing from 1.29 -> 1.28? Actually ^ will take the latest minor version anyway, which is 1.29. Better to leave it there. Minor point.